src/utils/oidc/authorize.ts

```diff
@@ -81,6 +81,7 @@ export const completeOidcLogin = async (
     homeserverUrl: string;
     identityServerUrl?: string;
     accessToken: string;
+    refreshToken?: string;
```

Comment:
documentation please. this is not mentioned in the doc-comment, unlike the other properties.

what does it mean if it is undefined?